vue/src/store/modules/workspace/mutations.js

```diff
@@ -0,0 +1,51 @@
+import {
+  SHOW_WORKSPACE_MODAL,
+  HIDE_WORKSPACE_MODAL,
+  GET_USER_WORKSPACES,
+  WORKSPACE_DELETED,
+  GET_BREAD_CRUMB
+} from './mutation-types';
+
+export default {
+  /**
+   *
+   * @param state
+   * @param workspace
+   */
+  [SHOW_WORKSPACE_MODAL](state, workspace) {
+    state.showModal = true;
+    state.modalWorkspace = workspace;
+  },
+  /**
+   *
+   * @param state
+   */
+  [HIDE_WORKSPACE_MODAL](state) {
+    state.showModal = false
```

Comment:
@skavt It is better to clear `state.modalWorkspace` when the modal hide vuex function is called. For future safety.